packages/server/modules/shared/repositories/roles.ts

```diff
@@ -0,0 +1,22 @@
+import { GetRoles, UpsertRole } from '@/modules/shared/domain/rolesAndScopes/operations'
+import { UserRole } from '@/modules/shared/domain/rolesAndScopes/types'
+import { Knex } from 'knex'
+
+let roles: UserRole[]
+
+export const getRoles =
```

Comment:
https://www.notion.so/speckle/General-JavaScript-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#b590a340a8474cb2aadf0e1bf6fe86fa

goes for both functions